Cargo.toml

```diff
@@ -39,7 +39,7 @@ rand_core = { version = "0.5", default-features = false }
 # `serde` is renamed to `our_serde` in order to avoid a name collision between
 # importing the serde dependency and enabling the curve25519-dalek/serde feature
 our_serde = { package = "serde", version = "1", default-features = false, optional = true, features = ["derive"] }
-zeroize = { version = "=1.3", default-features = false, features = ["zeroize_derive"] }
+zeroize = { version = ">=1.3", default-features = false, features = ["zeroize_derive"] }
```

Comment:
This is not the correct way to specify this requirement.

`>=1.3` will match *all* future versions including 2.0, 3.0, etc which may include semver-breaking changes.

IMO the requirement should just be changed to `1`

Reply:
Agreed.